kvsqlite/sqlite.py

```diff
@@ -150,6 +155,9 @@ def __connect(self):
         try:
             connection.execute("PRAGMA journal_mode = {}".format(self.journal_mode))
             connection.execute("PRAGMA synchronous = {}".format(self.synchronous))
+            connection.execute("PRAGMA temp_store = MEMORY")
+            connection.execute("PRAGMA cache_size = -64000")
+            connection.execute("PRAGMA mmap_size = 30000000000")
```

Comment:
These lines will be added/improved by me later. For now it's better to remove it.